### PR TITLE
Issues #263: Fix refresh on `view` when `remark` and `edit`

### LINKED ISSUE #263: Fix refresh bug on `view` when executing `remark` or `edit`

### LINKED ISSUE #264: Fix refresh bug on `view` when executing `remark` or `edit`
filtered interview schedule refreshes unecessarily when editing remark of a candidate or editing a candidate:
1. view today
2. remark 1 r/change remark OR edit 1 n/change name
Screenshot 2022-03-31 at 12 23 53 PM

suggested fix:
remove this line code from **BOTH** `remarkCommand` and `editCommand` files

Screenshot 2022-03-31 at 12 26 33 PM

_Originally posted by @charmainehly in https://github.com/AY2122S2-CS2103-F11-2/tp/issues/240#issuecomment-1084060852_

### PR DESCRIPTION
**Pull request by charmainehly**
Includes 

1. temporary fix to profile photo colour as the bug disallows me from loading up any profiles ( @LeeEnEn see issue tagged in the bug thread)
2. slight updates to documentation of UG
3. allow wrapping of result display box

@lzan98 appreciate your help to confirm that the deletion of the lines from `RemarkCommand` and `EditCommand` doesnt affect any behaviour from testings that you have done? - as from my understanding the setCandidate() method should not update the indexes of the candidates etc.

Closes #264

### CODE REVIEW

**Comment by lzan98** [comment]
@charmainehly as of now testing does not break after the deletion of the lines in `EditCommand` and `RemarkCommand`

File: docs/UserGuide.md [hunk 1]
```diff
@@ -173,18 +173,17 @@ Format: `list`
 
 Finds and lists candidates whose attribute field(s) contain(s) any of the given keyword(s).
 
-Format: `find k/KEYWORD [k/MORE_KEYWORDS]... f/ATTRIBUTE_FIELD`
+Format: `find k/KEYWORD [k/MORE_KEYWORDS]... [f/ATTRIBUTE_FIELD]`
```
Comment (domlimm) [inline docs/UserGuide.md:176]: Nice! En En and I were talking about this after we ended the meeting.

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -184,7 +184,8 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
 
     @Override
     public void start(Stage primaryStage) throws CommandException {
-        logger.info("Starting AddressBook " + MainApp.VERSION);
+        logger.info("Starting TalentAssistant " + MainApp.VERSION);
```
Comment (domlimm) [inline src/main/java/seedu/address/MainApp.java:187]: Maybe this? 😛 
```suggestion
        logger.info("Starting TalentAssistant™ " + MainApp.VERSION);
```

File: src/main/resources/view/ResultDisplay.fxml [hunk 3]
```diff
@@ -5,5 +5,5 @@
 
 <StackPane fx:id="placeHolder" styleClass="pane-with-border" xmlns="http://javafx.com/javafx/8"
     xmlns:fx="http://javafx.com/fxml/1">
-  <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display"/>
+  <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display" wrapText="true"/>
```
Comment (domlimm) [inline src/main/resources/view/ResultDisplay.fxml:8]: Really smart on wrapping the text for the feedback textbox 👍🏻

**Review by domlimm — CHANGES_REQUESTED** [review]
All LGTM! @charmainehly please take a look at the tiny miny minute little small suggestion!

**Comment by charmainehly** [comment]
> All LGTM! @charmainehly please take a look at the tiny miny minute little small suggestion!

Hi @domlimm lets change the `TalentAssistant™` occurrence in the next PR of next commit, as there are several occurrences within the same files, and some of them refer to the `TalentAssistant.json` that is being created.

**Review by domlimm — APPROVED** [review]
Noted on it @charmainehly! Approved!

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 4]
```diff
@@ -100,7 +98,6 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.setCandidate(candidateToEdit, editedCandidate);
-        model.updateFilteredCandidateList(PREDICATE_SHOW_ALL_CANDIDATES);
```
Comment (lzan98) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:103]: Thanks for making the change to fix the bug! 😄 After merging will try to do more extensive testing

**Review by lzan98 — APPROVED** [review]
LGTM 👍 Thanks for making the changes

**Pull request merged**